vertica/tests/test_unit.py

```diff
@@ -88,3 +89,30 @@ def test_client_logging_disabled(aggregator, instance):
             connection_load_balance=mock.ANY,
             connection_timeout=mock.ANY,
         )
+
+
+def test_client_logging_enabled_debug_if_agent_uses_debug(aggregator, instance):
```

Comment:
I think we need a test case with agent log level INFO and assert that log_level is None for vertica connect. (you can probably parametrize the existing test)
Otherwise, looks good.